docs/tutorials/using_ray_with_the_ensemble.ipynb

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
<p>I believe it would be valuable to add a sentence or two about use of the explicit options<code>use_map=False</code>  here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Would adding a comment inline be sufficient? We have a section on this in the docs here: https://tape.readthedocs.io/en/latest/tutorials/scaling_to_large_data.html#Data-Partitioning-and-Parallelization

Reply:
Comment inline or mentioning that it is elaborated more elsewhere

Reply:
Inline comment was added